index.js

```diff
@@ -123,19 +100,14 @@ async function processPullRequest (pullRequest, context) {
     .filter(async fileJSON => fileJSON !== 'deleted')
     .map(async fileJSON => {
       const filename = fileJSON.filename
-      const status = fileJSON.status
 
       const headRevision = apiHelper.getContents(context, filename, ref)
 
-      if (config.compareAgainstBaseline && status === 'modified') {
-        const baseRevision = apiHelper.getContents(context, filename, baseRef)
-        cache.saveFile(repoID, id, 'base', filename, (await baseRevision).data, 'python')
-      }
-
+      // TODO: merge this code with linter-specific path resolution
```

Comment:
Is this a TODO for a future PR or future commit on this PR?

Reply:
Future PR